tsk/fs/fs_attr.c

```diff
@@ -275,7 +275,7 @@ tsk_fs_attr_set_str(TSK_FS_FILE * a_fs_file, TSK_FS_ATTR * a_fs_attr,
 uint8_t
 tsk_fs_attr_set_run(TSK_FS_FILE * a_fs_file, TSK_FS_ATTR * a_fs_attr,
     TSK_FS_ATTR_RUN * a_data_run_new, const char *name,
-    TSK_FS_ATTR_TYPE_ENUM type, uint16_t id, TSK_OFF_T size,
+    TSK_FS_ATTR_TYPE_ENUM type, int16_t id, TSK_OFF_T size,
```

Comment:
Does this actually help? The underlying bits should still be the same. Are we comparing it's size to 0 at some point?

I thought the issue was some things thought it was a 32-bit value and some thought it was 16.... Am I mis-remembering? :-)